File: output/ChatGPT/avatar/Java/Python/codeforces_653_A.py
n = int(input())
a = []
for _ in range(n):
    value = int(input())
    a.append(value)
a = list(set(a))
a.sort()
found = False
for i in range(len(a)):
    if i + 1 < len(a) and i + 2 < len(a):
        if a[i] + 1 == a[i + 1] and a[i + 1] + 1 == a[i + 2]:
            found = True
print("YES" if found else "NO")

File: output/ChatGPT/avatar/Java/C++/codeforces_653_A.cpp
#include <iostream>
#include <vector>
#include <algorithm>

int main() {
    int n;
    std::cin >> n;
    std::vector<int> a(n);
    
    for (int i = 0; i < n; i++) {
        std::cin >> a[i];
    }
    
    std::sort(a.begin(), a.end());
    a.erase(std::unique(a.begin(), a.end()), a.end());
    
    bool found = false;
    for (int i = 0; i < a.size(); i++) {
        if (i + 1 < a.size() && i + 2 < a.size()) {
            if (a[i] + 1 == a[i + 1] && a[i + 1] + 1 == a[i + 2]) {
                found = true;
            }
        }
    }
    
    std::cout << (found ? "YES" : "NO") << std::endl;
    
}